src/plugins/saved_objects_management/public/management_section/mount_section.tsx

```diff
@@ -0,0 +1,206 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { HashRouter, Switch, Route, useParams, useLocation } from 'react-router-dom';
+import { parse } from 'query-string';
+import { get } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { I18nProvider } from '@kbn/i18n/react';
+import { CoreSetup, CoreStart, ChromeBreadcrumb, Capabilities } from 'src/core/public';
+import { ManagementAppMountParams } from '../../../management/public';
+import { DataPublicPluginStart } from '../../../data/public';
+import { StartDependencies, SavedObjectsManagementPluginStart } from '../plugin';
+import {
+  ISavedObjectsManagementServiceRegistry,
+  SavedObjectsManagementActionServiceStart,
+} from '../services';
+import { SavedObjectsTable } from './objects_table';
+import { SavedObjectEdition } from './object_view';
+import { getAllowedTypes } from './../lib';
+
+interface MountParams {
+  core: CoreSetup<StartDependencies, SavedObjectsManagementPluginStart>;
+  serviceRegistry: ISavedObjectsManagementServiceRegistry;
+  mountParams: ManagementAppMountParams;
+}
+
+export const mountManagementSection = async ({
+  core,
+  mountParams,
+  serviceRegistry,
+}: MountParams) => {
+  const [coreStart, { data }, pluginStart] = await core.getStartServices();
+  const { element, basePath, setBreadcrumbs } = mountParams;
+  const allowedTypes = await getAllowedTypes(coreStart.http);
+  const capabilities = coreStart.application.capabilities;
+
+  ReactDOM.render(
+    <I18nProvider>
+      <HashRouter basename={basePath}>
+        <Switch>
+          <Route path={'/:service/:id'} exact={true}>
+            <RedirectToHomeIfUnauthorized capabilities={capabilities}>
+              <SavedObjectsEditionPage
+                coreStart={coreStart}
+                serviceRegistry={serviceRegistry}
+                setBreadcrumbs={setBreadcrumbs}
+              />
+            </RedirectToHomeIfUnauthorized>
+          </Route>
+          <Route path={'/'} exact={false}>
+            <RedirectToHomeIfUnauthorized capabilities={capabilities}>
+              <SavedObjectsTablePage
+                coreStart={coreStart}
+                dataStart={data}
+                serviceRegistry={serviceRegistry}
+                actionRegistry={pluginStart.actions}
+                allowedTypes={allowedTypes}
+                setBreadcrumbs={setBreadcrumbs}
+              />
+            </RedirectToHomeIfUnauthorized>
+          </Route>
+        </Switch>
+      </HashRouter>
+    </I18nProvider>,
+    element
+  );
+
+  return () => {
+    ReactDOM.unmountComponentAtNode(element);
+  };
+};
+
+const RedirectToHomeIfUnauthorized: React.FunctionComponent<{
+  capabilities: Capabilities;
+}> = ({ children, capabilities }) => {
+  const allowed = capabilities?.management?.kibana?.objects ?? false;
+  if (!allowed) {
+    window.location.hash = '/home';
```

Comment:
Yea it is. Once `home` is migrated outside of the legacy `kibana`, this won't no longer work anyway, and this will be changed for a `application.navigateToApp` call.

Reply:
Why is `application.navigateToApp` unavailable for changing the url? 

Capabilities should really have an observable API so the plugin could be disabled pre render. I created a ticket here - https://github.com/elastic/kibana/issues/63115

Reply:
@mattkime home isn't a real application (in the sense of `core.application`) yet, it's just a route in the legacy kibana app (using angular routing).

This will change once https://github.com/elastic/kibana/issues/61304 is done